<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cyx_Base_Window</class>
 <widget class="QWidget" name="Cyx_Base_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cyx_Base_Window</string>
  </property>
  <layout class="QHBoxLayout" name="h_main_layout">
   <item>
    <widget class="QWidget" name="content_widget" native="true">
     <property name="styleSheet">
      <string notr="true">#content_widget{
background-color: rgb(255, 255, 255);
border-radius:5px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="v_content_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="h_top_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="title_label">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;Adobe Garamond Pro&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="min_btn">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	background-color: rgb(233, 234, 226);
	image:url(D:/git_Workspace/windows_screen/windows_screen/Resources/min.png);
}

QPushButton:hover{
	background-color: rgb(214, 211, 209);
}

QPushButton:pressed{
	background-color: rgb(233, 234, 226);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="max_btn">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	background-color: rgb(234, 230, 228);
	image:url(D:/git_Workspace/windows_screen/windows_screen/Resources/max.png);
}

QPushButton:hover{
	background-color: rgb(214, 211, 209);
}

QPushButton:pressed{
	background-color: rgb(234, 230, 228);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_btn">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	border-top-right-radius:5px;
	background-color: rgb(255, 130, 58);
	image:url(D:/git_Workspace/windows_screen/windows_screen/Resources/close.png);
}

QPushButton:hover{
	background-color: rgb(227, 75, 0);
}

QPushButton:pressed{
	background-color: rgb(255, 153, 69);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="thrid_widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
